libs/remote_storage/tests/test_real_azure.rs

```diff
@@ -623,3 +607,32 @@ async fn upload_simple_azure_data(
         ControlFlow::Continue(uploaded_blobs)
     }
 }
+
+// FIXME: copypasted from test_real_s3, can't remember how to share a module which is not compiled
```

Comment:
tbh the entire `test_real_azure.rs` bases on a copy pasted version of test_real_s3, so it's not really *that* special here in that regard. one could share way more than just this function (that's what "see if we can share more between S3's test and azure's test files" in https://github.com/neondatabase/neon/pull/5446 is about).